sippy-ng/src/App.js

```diff
@@ -460,9 +460,14 @@ export default function App(props) {
                     />
 
                     <Route
-                      path="/job_runs/:jobrunid/intervals"
+                      path="/job_runs/:jobrunid/:jobname/:repoinfo/:pullnumber/intervals"
```

Comment:
This looks busted for non-presubmits where you only specify jobname? Does it still work if the others are missing? It makes me nervous either way unless the behavior being relied on is documented. Could we do query params here as well? Curious what @stbenjam would like, we're trying to keep backward compatibility, but also accept and use new info.

Reply:
Making them optional allows us to not pass them and still use the route.